src/test/java/bio/terra/common/logging/LoggingTest.java

```diff
@@ -200,6 +201,34 @@ public void testStructuredLogging(CapturedOutput capturedOutput) {
     assertThat((String) readJson(event4, "$.foo.bar")).isEqualTo("baz");
   }
 
+  @Test
+  public void testAlertLogging(CapturedOutput capturedOutput) {
+    ResponseEntity<String> response =
+        testRestTemplate.getForEntity("/testAlertLogging", String.class);
+    assertThat(response.getStatusCode().value()).isEqualTo(200);
+
+    String[] lines = capturedOutput.getAll().split("\n");
+    String event1 = null;
+    String event2 = null;
+    for (String line : lines) {
+      String message = readJson(line, "$.message");
+      if (message != null && message.contains("test alert message")) {
```

Comment:
we should extract a package visible filelds for the error messages to share between the two test classes.

Reply:
pulled this logic into `getLogContainingMessage` for all tests to use